superset/models/reports.py

```diff
@@ -171,6 +172,7 @@ class ReportExecutionLog(Model):  # pylint: disable=too-few-public-methods
 
     __tablename__ = "report_execution_log"
     id = Column(Integer, primary_key=True)
+    uuid = Column(UUIDType(binary=True))
```

Comment:
Without the use of celery's UUID, you can just use `default=uuid.uuid4`

We probably should think about moving out of sqlalchemy_utils...

Reply:
We're already pretty tied to it - this doesn't tie us any deeper. I think it's OK for now.